packages/atlas/lib/src/marker.dart

```diff
@@ -45,3 +48,14 @@ class Marker {
   int get hashCode =>
       id.hashCode ^ position.hashCode ^ icon.hashCode ^ zIndex.hashCode;
 }
+
```

Comment:
I think you need to add the {@template anchor} and other docs from my comment here in order for the @macro to work

Reply:
i fix that